### DISCUSSION TITLE
Проблема с выбором автора ranobelib

### DISCUSSION THREAD

**Discussion by AVolkhv** [OnlyFart/Elib2Ebook#87] [Q&A] [answered]
[https://ranobelib.me/elder-blood-witcher?section=info&ui=6618335](url)
Подскажите, как выбрать первого автора (перевода). в хелпе не нашел как, программа все время цепляется за второго, "--start 1 --end 165" в данном случае не помогают, проблем бы не возникло, да и вопроса то же, еслиб второй не начал со 111 главы. и получается первую часть невозможно скачать, или я не врубился в команды.

**Comment by OnlyFart** [comment, marked as answer]
Вот здесь есть ответ на ваш вопрос https://github.com/OnlyFart/Elib2Ebook/issues/18

**Comment by AVolkhv** [comment]
> Вот здесь есть ответ на ваш вопрос #18
Благодарю за ответ. Не работало потому что вставлял цельную ссылку [https://ranobelib.me/elder-blood-witcher?bid=15095&section=chapters&ui=6618335](url) и программа ее не воспринимала, после удаления всего до bid=15095, все заработало.
